src/content/docs/en/guides/markdown-content.mdx

```diff
@@ -397,7 +397,7 @@ When using the frontmatter `layout` property, you must include the `<meta charse
 
 ## Fetching Remote Markdown
 
-Astro does not include built-in support for remote Markdown outside of [content collections](/en/guides/content-collections/).
+Astro does not include built-in support for remote Markdown.
 
 To fetch remote Markdown directly and render it to HTML, you will need to install and configure your own Markdown parser from NPM. This will not inherit from any of Astro's built-in Markdown settings that you have configured.
```

Comment:
```suggestion
Astro's internal Markdown processor is not available for processing remote Markdown.

To fetch remote Markdown for use in [content collections](/en/guides/content-collections), you can [build a custom loader](/en/guides/content-collections/#building-a-custom-loader) with access to a [`renderMarkdown()` function](/en/reference/content-loader-reference/#rendermarkdown).

To fetch remote Markdown directly and render it to HTML, you will need to install and configure your own Markdown parser from NPM. This will not inherit from any of Astro's built-in Markdown settings that you have configured.
```
OK, might have gone link-happy here, but what do we think about something like this @paul-sachs ? (Until such time as we actually do expose the internal Markdown parser?)